src/assemble/bindist-mapr.xml

```diff
@@ -121,6 +121,13 @@
         <include>xml-apis:xml-apis:jar:*</include>
         <include>com.thoughtworks.xstream:xstream:jar:*</include>
         <include>junit:junit:jar:*</include>
+        <include>com.fasterxml.woodstox:woodstox-core:jar:*</include>
```

Comment:
For the mapr build, we are still using Hadoop 2.7.0-mapr-1602, all the new dependencies we added for Hadoop 3.3.0 are not needed here.

One further question: could you please check with Shiva whether we are still releasing mlcp MapR build? I think we have discontinued the support. It may make sense to refactor the pom files to get rid of MapR build when we have the bandwidth.

Thanks,
Vanessa